lib/samples/manage_features/manage_features.dart

```diff
@@ -0,0 +1,433 @@
+      _damageServiceFeatureTable = table;
+      // Load the table.
+      await _damageServiceFeatureTable.load();
+      // Get the required field from the table - in this case, damage type.
+      final damageTypeField =
+          table.fields.firstWhere((field) => field.name == damageTypeFieldName);
+      // Get the domain for the field.
+      final domain = damageTypeField.domain as CodedValueDomain;
+      // Update the dropdown menu with the attribute values from the domain.
+      configureAttributeDropdownMenuItems(domain);
+
+      // Create a feature layer to visualize the features in the table.
+      _damageFeatureLayer =
+          FeatureLayer.withFeatureTable(_damageServiceFeatureTable);
+      // Create a map with the ArcGIS Streets basemap.
+      final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISStreets);
+      // Add the feature layer to the map's operational layers.
+      map.operationalLayers.add(_damageFeatureLayer);
+      // Set the map to the map view controller.
+      _mapViewController.arcGISMap = map;
+      // Zoom to an initial viewpoint.
+      _mapViewController.setViewpoint(
+        Viewpoint.fromCenter(
+          ArcGISPoint(
+            x: -10800000,
+            y: 4500000,
+            spatialReference: SpatialReference.webMercator,
+          ),
+          scale: 3e7,
+        ),
+      );
+      // Set the ready state variable to true to enable the sample UI.
+      setState(() => _ready = true);
+    } else {
+      showMessageDialog('Unable to access the required feature table.');
+    }
+  }
+
+  void onTap(Offset localPosition) async {
+    // Configure actions when a user taps on the map, depending on the selected operation.
+    if (_selectedOperation == FeatureManagementOperation.create) {
+      // Create a feature if create is selected.
+      createFeature(localPosition);
+    } else if (_selectedOperation == FeatureManagementOperation.geometry &&
+        _selectedFeature != null) {
+      // If update geometry is selected, update the selected feature.
+      updateGeometry(_selectedFeature!, localPosition);
+    } else {
+      // Otherwise attempt to identify and select a feature.
+      identifyAndSelectFeature(localPosition);
+    }
+  }
+
+  void createFeature(Offset localPosition) async {
+    // Disable the UI while the async operations are in progress.
+    setState(() => _ready = false);
+
+    // Create the feature.
+    final feature = _damageServiceFeatureTable.createFeature();
+
+    // Get the normalized geometry for the tapped location and use it as the feature's geometry.
+    final geometry = _mapViewController.screenToLocation(screen: localPosition);
+    if (geometry != null) {
+      final normalizedGeometry =
+          GeometryEngine.normalizeCentralMeridian(geometry);
+      feature.geometry = normalizedGeometry;
+
+      // Set feature attributes.
+      feature.attributes['typdamage'] = 'Minor';
+      feature.attributes['primcause'] = 'Earthquake';
```

Comment:
We have `damageTypeFieldName` we can use instead of `'typdamage'`, but we don't have a variable for `'primcause'`.

I'm generally of the opinion that defining a member variable like this is more trouble than it's worth. Like in this case if we needed to change `damageTypeFieldName` for some reason, we'd still have to do a global search-and-replace for errant hardcoded uses of `'typdamage'` just to be sure. We could add a member variable for `'primcause'`, and then add one for `'objectid'`, but that seems weird. And it also seems weird to have an exception just for `'objectid'`.

Reply:
Removed the variable